lib/ActivityStream.jsm

```diff
@@ -239,6 +239,7 @@ const PREFS_CONFIG = new Map([
         collapsible: true,
         enabled: isEnabled,
         show_spocs: showSpocs({geo}),
+        hardcoded_layout: false,
         personalized: false,
         // This is currently an exmple layout used for dev purposes.
         layout_endpoint: "https://getpocket.cdn.mozilla.net/v3/newtab/layout?version=1&consumer_key=$apiKey&layout_variant=basic",
```

Comment:
Do we want to keep both values (layout_endpoint and hardcoded_layout) vs just having layout_endpoint be a special value. Or perhaps even if there's no layout_endpoint defined, assume it's the default hardcoded layout object.

Reply:
Went through a few ideas on this, ended up picking this one because it makes it easier to toggle between them without having to remember or reset the url endpoint value constantly.

Reply:
Not a huge issue. Seems like it could introduce some testing confusion "I set the endpoint but nothing changed" If this is for developer convenience, there is about:newtab#devtools-ds to switch some variants.

Reply:
Although changing the url to something special would then break that devtools behavior 😉

Reply:
Yeah I considered the test confusion too but this felt more explicit.